test/e2e/feature/features.go

```diff
@@ -0,0 +1,24 @@
+package feature
+
+import "github.com/ovn-org/ovn-kubernetes/test/e2e/label"
+
+var (
+	Service                     = label.NewFeature("Service")
+	NetworkPolicy               = label.NewFeature("NetworkPolicy")
+	AdminNetworkPolicy          = label.NewFeature("AdminNetworkPolicy")
+	BaselineNetworkPolicy       = label.NewFeature("BaselineNetworkPolicy")
+	NetworkSegmentation         = label.NewFeature("NetworkSegmentation")
+	EgressIP                    = label.NewFeature("EgressIP")
+	EgressService               = label.NewFeature("EgressService")
+	EgressFirewall              = label.NewFeature("EgressFirewall")
+	EgressQos                   = label.NewFeature("EgressQos")
+	ExternalGateway             = label.NewFeature("ExternalGateway")
+	DisablePacketMTUCheck       = label.NewFeature("DisablePacketMTUCheck")
+	KubeVirt                    = label.NewFeature("KubeVirt")
```

Comment:
KubeVirt is not a feature, it is an upstream project. 

I'd name this `Virt` or `Virtualization` or something along those lines.